crates/uv/src/settings.rs

```diff
@@ -376,6 +376,7 @@ impl PythonInstallSettings {
 #[derive(Debug, Clone)]
 pub(crate) struct PythonUninstallSettings {
     pub(crate) targets: Vec<String>,
+    pub(crate) all: bool,
```

Comment:
We should mark these as conflicting right?

Reply:
Omg sorry I'm dumb I looked at the settings struct. Bad review comments...

Reply:
Lol appreciate you!